website/templates/emails/new_pending_withdraw_requests.html.mako

```diff
@@ -0,0 +1,21 @@
+<table class="comment-row" border="0" cellpadding="8" cellspacing="0" width="100%" align="center" style="font-size: 13px;background: #fff;border: 1px solid #eee;border-radius: 5px;margin-bottom: 10px;padding: 0px !important;">
```

Comment:
Read this and get Product language

Reply:
Do we still need something here?

Reply:
So this was to add digests for withdraw requests but that's not in the reqs so I'm deleting it.